src/components/panel/panel.js

```diff
@@ -1311,6 +1328,31 @@ MdPanelRef.prototype._configureClickOutsideToClose = function() {
 
 
 /**
+ * Configures the listeners for updating the panel position on scroll.
+ * @private
+*/
+MdPanelRef.prototype._configureScrollListener = function() {
+  var updatePosition = angular.bind(this, this._updatePosition);
+  var debouncedUpdatePosition = this._$$rAF.throttle(updatePosition);
+  var self = this;
+
+  var onScroll = function() {
+    if (!self._config['disableParentScroll']) {
+      debouncedUpdatePosition();
+    }
+  };
+
+  // Add listeners.
+  this._$window.addEventListener('scroll', onScroll, true);
+
+  // Queue remove listeners function.
+  this._removeListeners.push(function() {
+    self._$window.removeEventListener('scroll', onScroll);
```

Comment:
I think you are missing an argument. If you use the *capture* phase in the `addEventListener( )`, then you must match when removing:

```js
self._$window.removeEventListener('scroll', onScroll, true);
```